charts/nginx-gateway-fabric/values.yaml

```diff
@@ -462,6 +462,41 @@ nginx:
   #     minimum: 1
   #     maximum: 65535
   #     description: The number of worker connections for NGINX. Default is 1024.
+  #   dnsResolver:
+  #     type: object
+  #     description: DNSResolver specifies the DNS resolver configuration for external name resolution. This enables support for routing to ExternalName Services.
+  #     properties:
+  #       addresses:
+  #         type: array
+  #         description: List of DNS server addresses. Each address specifies a type and value.
+  #         items:
+  #           type: object
+  #           properties:
+  #             type:
+  #               type: string
+  #               enum:
+  #                 - IPAddress
+  #                 - Hostname
+  #               description: Type specifies the type of address.
+  #             value:
+  #               type: string
+  #               description: Value specifies the address value.
```

Comment:
Should probably have min and max length here too for consistency.

Reply:
The min and the max is on line 487 for the `addresses` type, I don't think it makes sense to be on the value field?

Reply:
:facepalm: yes of course